### PR TITLE
fixing config location for ubuntu package install

### PR DESCRIPTION
**Pull request by heckj**
- thanks George!

### CODE REVIEW

**Comment by heckj** [comment]
FYI @gpaulos

**Comment by tldavies** [comment]
:+1:

File: docs/rackhd/ubuntu_package_installation.rst [hunk 1]
```diff
@@ -90,7 +90,7 @@ Touch those files to allow the upstart scripts to start automatically.
 
 **RACKHD APPLICATIONS**
 
-Created the required file /opt/onrack/etc/config.json , you can use the demonstration
+Created the required file /opt/monorail/config.json , you can use the demonstration
```
Comment (johren) [inline docs/rackhd/ubuntu_package_installation.rst]: While you're in here...should "Created" be "Create"?

Comment (heckj) [inline docs/rackhd/ubuntu_package_installation.rst]: DOH! Yep, will update!

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**